test-requirements.txt

```diff
@@ -1,5 +1,3 @@
-black==23.3.0
-flake8
 mock
 pexpect
 yamllint
```

Comment:
`yamllint` is also not needed.

Reply:
You can remove `mock` and `coverage` too, I believe.

Reply:
mock we need for unit tests